src/docs/resources/faq.md

```diff
@@ -194,7 +194,7 @@ work.
 
 Debug mode builds use a virtual machine (VM) to run Dart code
 (hence the "debug" banner they show to remind people that they're
-slightly slower) in order to enable stateful hot reload.
+slightly slower) in order to enable Stateful Hot Reload.
```

Comment:
Here it does seem like it's talking about an Official Feature.

P.S. I would change "they show" to "shown".  And I'd probably change "that they're slightly slower" to "that these builds are slightly slower."

Reply:
Great, I'll leave this as-is, I want to keep this PR focused our use of `stateful hot reload`